plugins/postgres/dbt/include/postgres/macros/adapters.sql

```diff
@@ -4,14 +4,27 @@
 
   {{ sql_header if sql_header is not none }}
 
-  create {% if temporary -%}
-    temporary
-  {%- elif unlogged -%}
-    unlogged
-  {%- endif %} table {{ relation }}
-  as (
-    {{ sql }}
-  );
+  {% if config.get('constraints_enabled', False) %}
```

Comment:
The initial piece (lines 8-12) is the same for both conditions. This if block should be moved down to just the piece that is the difference.

Reply:
done